modules/exploits/openbsd/local/xorg_x11_suid_server.rb

```diff
@@ -0,0 +1,171 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
```

Comment:
Considering this clobber files, I don't think it should necessarily be `ExcellentRanking`.

https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking

Reply:
You may want to consider https://github.com/rapid7/metasploit-framework/wiki/Definition-of-Module-Reliability,-Side-Effects,-and-Stability instead.

Reply:
I'd at least leave a note suggesting files will be clobbered. I'm glad you implemented a backup routine!

Reply:
While a backup routine is a good idea, it's worth noting that the vulnerable functionality also creates a backup by default (`<filename>.old`), hence the need for manual cleanup.